### PR TITLE
Added DBMS Book

### PR DESCRIPTION
**Pull request by YashSalunke12**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-subjects.md [hunk 1]
```diff
@@ -255,6 +255,7 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [Readings in Database Systems, 5th Ed.](http://www.redbook.io)
 * [Temporal Database Management](http://people.cs.aau.dk/~csj/Thesis/) - Christian S. Jensen
 * [The Theory of Relational Databases](http://web.cecs.pdx.edu/~maier/TheoryBook/TRD.html)
+* [Database Management Systems/Managing Database](https://ebooks.lpude.in/management/mba/term_3/DCAP204_MANAGING_DATABASE_DCAP402_DATABASE_MANAGEMENT_SYSTEMS.pdf) - Dr Anil Sharma (PDF)
```
Comment (shrn01) [inline books/free-programming-books-subjects.md:258]: @YashSalunke12 is the book free to be used by people outside LPU?

Comment (YashSalunke12) [reply books/free-programming-books-subjects.md:258]: @shrn01 bro can you tell me, what should I change in this? sorry but this is my first time contributing an project! why build failed?

Comment (shrn01) [reply books/free-programming-books-subjects.md:258]: Hey don't worry. Let me help you out.

Comment (shrn01) [inline books/free-programming-books-subjects.md:258]:
```suggestion
* [Database Management Systems/Managing Database](https://ebooks.lpude.in/management/mba/term_3/DCAP204_MANAGING_DATABASE_DCAP402_DATABASE_MANAGEMENT_SYSTEMS.pdf) - Anil Sharma (PDF)
```

**Review by shrn01 — CHANGES_REQUESTED** [review]

File: books/free-programming-books-subjects.md (hunk 1 above)
Comment (YashSalunke12) [reply books/free-programming-books-subjects.md]: @shrn01  Done

Comment (YashSalunke12) [reply books/free-programming-books-subjects.md]: @eshellman Changes done

Comment (YashSalunke12) [reply books/free-programming-books-subjects.md]: @shrn01 Changes done

**Comment by eshellman** [comment]
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.

File: books/free-programming-books-subjects.md [hunk 2]
```diff
@@ -255,6 +255,7 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [Readings in Database Systems, 5th Ed.](http://www.redbook.io)
 * [Temporal Database Management](http://people.cs.aau.dk/~csj/Thesis/) - Christian S. Jensen
 * [The Theory of Relational Databases](http://web.cecs.pdx.edu/~maier/TheoryBook/TRD.html)
+* [Database Management Systems/Managing Database](https://ebooks.lpude.in/management/mba/term_3/DCAP204_MANAGING_DATABASE_DCAP402_DATABASE_MANAGEMENT_SYSTEMS.pdf) - Anil Sharma (PDF)
```
Comment (bugbunnyy) [inline books/free-programming-books-subjects.md:258]: Move your changes after this course line
`[Database Fundamentals]`

Comment (shrn01) [reply books/free-programming-books-subjects.md:258]: @YashSalunke12 please make this change

**Pull request closed without merging**